docs/README.md

```diff
@@ -3,7 +3,7 @@
   <br />
   <img src="https://user-images.githubusercontent.com/1093738/101672561-06aa7480-3a24-11eb-89d1-787fa7112138.png" alt="XState" width="150"/>
   <br />
-    <sub><strong>State machines and statecharts for the modern web.</strong></sub>
+    <sub><strong>Handle complexity anywhere JavaScript runs</strong></sub>
```

Comment:
Ok, so I agree with your point about “for the modern web” being limited, but here we miss the specificity of state machines and statecharts, and “handle complexity” is perhaps too unspecific for a docs landing page. (More of an overall landing page vibe.)

How about using your line as below: “Handle complexity anywhere JavaScript runs using statecharts.” (Just revised as I prefer the wording you used.)

Then we’re:
- *handle complexity*: WHY - benefit of statecharts/XState
- *with statecharts*: HOW - what we’re using to handle the complexity, just statecharts because its technically redundant to say “state machines and statecharts”, and XState is always statecharts and usually more than state machines
- *anywhere JavaScript runs*: WHERE - where you can handle complexity

This isn’t a perfect solution:

- Do we need a WHAT? Specify that the complexity is in applications and/or state specifically?
- Does the length make it less of a tagline and more of just a sentence?

I’m only getting into mega detail on this because it’s the first line on the docs homepage 😊

Reply:
```suggestion
    <sub><strong>Build with statecharts, anywhere JavaScript runs</strong></sub>
```

⬆️ ❓

Reply:
Agree that the tagline should be more descriptive of what XState actually is

Reply:
Yep I'm down for this, anything that isn't 'modern web' is fine by me and we can workshop a proper tagline later